### PR TITLE
PCHR-634: Due date modified

### PR DESCRIPTION
**Pull request by kkalamarski**
## Problem:
Dates were sent to backend with timezone information, which caused bug we've already fixed in HRJobRoles. (Date was going back by 1 day)

If wrong date was saved to database, also notification email had wrong date.

## Solution:
Using `moment()` to parse date to format `YYYY-MM-DD`, which is correctly read by backend.
Now date is correctly saved to database, so date in email notification is also correct.

Also removed CommonJS version of `angular-date` and implemented `common.angularDate` (AMD).

### CODE REVIEW

File: uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/modules/run.js [hunk 1]
```diff
@@ -9,16 +9,22 @@ define([
     'tasks-assignments/vendor/text-angular',
     'tasks-assignments/modules/config',
     'tasks-assignments/modules/settings',
+    // TODO use only one router
```
Comment (AkA84) [inline uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/modules/run.js]: Why is the app using two different routers?

Comment (kkalamarski) [inline uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/modules/run.js]: I have no idea. It was like this when I started working on that. But I'll refactor it when I work on certain modules.

Comment (AkA84) [inline uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/modules/run.js]: Would it be a big refactoring? Could it be done as part of this PR?

Comment (kkalamarski) [inline uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/modules/run.js]: Actually it can be done.

**Comment by AkA84** [comment]
Approved

**Pull request merged**